README.md

```
@@ -80,7 +80,20 @@ pip install open-space-toolkit-core
 
 ## Documentation
 
-Documentation is available [here](https://open-space-collective.github.io/open-space-toolkit-core):
+Documentation is available [here](https://open-space-collective.github.io/open-space-toolkit-core).
```

markdownlint-cli2 (0.23.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)